includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -416,6 +416,77 @@ public function validate_recaptcha_field( $success ) {
 		return $success;
 	}
 
+	/**
+	 * Output the agreement checkbox field.
+	 *
+	 * @since 1.35.2
+	 */
+	public function display_agreement_checkbox_field() {
+		$terms_page_id      = get_option( 'job_manager_terms_page_id', false );
+		$conditions_page_id = get_option( 'job_manager_conditions_page_id', false );
+		$terms_label        = esc_html__( 'terms', 'wp-job-manager' );
+		$conditions_label   = esc_html__( 'conditions', 'wp-job-manager' );
+
+		// translators: %1$s is the terms page link; %2$s is the terms label.
+		$terms_placeholder = $terms_page_id ? sprintf(
```

Comment:
I've never seen the Terms and Conditions split between two pages. Is this a regional thing? WooCommerce has just one option for the `Terms and Conditions` page, for example.

Reply:
Ah, probably I deduced it by mistake because in Brazilian sites usually we added "Terms and Privacy Policy" with 2 different links. I fixed it here: https://github.com/Automattic/WP-Job-Manager/pull/2155/commits/bb17b09be08468d193a598e519d7e0d3b41ce274